approval_polls/views.py

```diff
@@ -522,3 +522,44 @@ def post(self, request, *args, **kwargs):
             return HttpResponseRedirect(
                 reverse('approval_polls:embed_instructions', args=(p.id,))
             )
+
+
+class EditView(generic.View):
+
+    @method_decorator(login_required)
+    def get(self, request, *args, **kwargs):
+
+        # TODO handle polls that don't exist.
+        poll = Poll.objects.get(id=kwargs['poll_id'])
+        choices = Choice.objects.filter(poll=kwargs['poll_id'])
+        return render(request, 'approval_polls/edit.html', {
+            'poll': poll,
+            'choices': choices,
+            'closedatetime': poll.close_date.strftime("%Y/%m/%d %H:%M") if poll.close_date else ""
+        })
+
+    @method_decorator(login_required)
+    def post(self, request, *args, **kwargs):
+        poll = Poll.objects.get(id=kwargs['poll_id'])
+        closedatetime = request.POST['close-datetime']
+        try:
+            closedatetime = datetime.datetime.strptime(
+                closedatetime,
+                '%Y/%m/%d %H:%M'
+            )
+            current_datetime = timezone.localtime(timezone.now())
+            current_tzinfo = current_datetime.tzinfo
+            closedatetime = closedatetime.replace(
+                tzinfo=current_tzinfo
+            )
```

Comment:
The original thought was to have the datetime dialog work in the users local timezone and convert that to UTC while storing it in the database. I'll need to check if that is not the case, since I'm in PDT as well, it seems to look correct to me.